test/resources/audioWorkletStream.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+
+<html>
+  <head> </head>
+
+  <body>
+    <h1>AudioWorklet Transform Stream</h1>
+    <script type="worklet">
+      class AudioWorkletStream extends AudioWorkletProcessor {
+        constructor(options) {
+          super();
+          console.log(this);
+          this.port.onmessage = e => {
+            this.writable = e.data;
+            this.writer = this.writable.getWriter();
+            this.port.postMessage('stream')
+          };
+        }
+        process(inputs, outputs) {
+          // convert to Int16Array, UintArray here
+          const channels = inputs.flat().reduce((f32, channel, index) => (f32.set(channel, !index ? index : 128), f32), new Float32Array(256));
+          this.writer.write(channels);
+          return true;
+        }
+      };
+      registerProcessor(
+        'audio-worklet-stream',
+        AudioWorkletStream
+      );
+    </script>
+    <script>
+      document.querySelector('h1').onclick = async () => {
+        const mediaStream = await navigator.mediaDevices.getUserMedia({
+          audio: true,
+        });
+        console.log(mediaStream);
+        const worklet = URL.createObjectURL(
+          new Blob(
+            [document.querySelector('script[type=worklet]').textContent],
+            {
+              type: 'text/javascript',
+            }
+          )
+        );
+        const ac = new AudioContext();
+        const source = new MediaStreamAudioSourceNode(ac, { mediaStream });
+        const { readable, writable } = new TransformStream();
+        const reader = readable.getReader();
+        reader.read().then(function processAudioStream({ value, done }) {
+          if (done) return reader.closed; // Promise
+          // do stuff with value from AudioWorklet
+          console.log(
+            value instanceof Float32Array,
+            value.length,
+            value.every((float) => float === 0),
+            done
+          );
```

Comment:
So, this is where we'd pass things along to the node.js-style stream, right?

Reply:
That is one location in the code to covert the `Float32Array`(s) from `AudioWorkletProcessor.process` to different values, for example, `Int16Array`, `Uint8Array`.

Another location is on the `readable` side which can be passed to `pipeThrough()` and `pipeTo()`, or `Response()`.  It is also possible to pass a `ReadableStream` to local or remote `quic-transport` server using `WebTransport`, where if necessary the data can be passed to node.js code.

I have little experience with node.js streams. I have experimented with Streams API https://streams.spec.whatwg.org/. If the API's are compatible, yes, that is one part of the code where the data as a `ReadableStream` can be passed to node.js.

Reply:
> So, this is where we'd pass things along to the node.js-style stream, right?

Yes, that is the data read in main thread from `AudioWorklet` output, logged at `console`.